flink-connector-hbase-2.2/src/test/java/org/apache/flink/connector/hbase2/HBaseConnectorITCase.java

```diff
@@ -66,16 +66,13 @@
 import static org.apache.flink.table.api.Expressions.$;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
 
 /** IT cases for HBase connector (including source and sink). */
-public class HBaseConnectorITCase extends HBaseTestBase {
+class HBaseConnectorITCase extends HBaseTestBase {
 
-    @ClassRule
-    public static final MiniClusterWithClientResource MINI_CLUSTER =
-            new MiniClusterWithClientResource(
+    @RegisterExtension
+    private static final MiniClusterExtension MINI_CLUSTER_EXTENSION =
```

Comment:
There is moving to `HBaseTestingClusterAutoStarter` already, maybe we can remove it.

Reply:
I checked the commit that added this (FLINK-24077) and it was introduced to fix a flaky test, so I went with the dummy approach to keep it as close to the state it was before. But since this code lives in a separate repo now and the underlying logic with JUnit5 might do not have that defect anyways I think it can be removed. If it introduces test flakiness again, we will know the probable source.